lua/starfall/libs_sv/prop.lua

```diff
@@ -430,9 +458,57 @@ function props_library.createSent(pos, ang, class, frozen)
 		end
 
 		hookcall = "PlayerSpawnedVehicle"
-
+	elseif sent2 then
+		if ply ~= NULL then
+			if (scripted_ents.GetStored(class).t.AdminOnly and not ply:IsAdmin()) then SF.Throw("This sent is admin only!", 2) end
+			if gamemode.Call("PlayerSpawnSENT", ply, class) == false then SF.Throw("Another hook prevented the sent from spawning", 2) end
+		end
+		
+		local enttbl = {}
+		data = data or {}
+		
+		-- Apply data
+		for param, org in pairs(sent2[1]) do
+			local value = data[param]
+			
+			if value then
+				value = ounwrap(value) or value
+				
+				checkluatype(value, org[2], nil, "Parameter: " .. param)
+				
+				enttbl[org[1]] = value
+			else
+				enttbl[org[1]] = org[3]
+			end
+		end
+		
+		-- Supply additional data
+		enttbl.Data = enttbl
+		enttbl.Name = ""
+		enttbl.Class = class
+		enttbl.Pos = pos
+		enttbl.Angle = ang
+		
+		if sent2._preFactory then
+			sent2._preFactory(ply, enttbl)
+		end
+		
+		entity = duplicator.CreateEntityFromTable(ply, enttbl)
+		
+		if sent2._postFactory then
+			sent2._postFactory(ply, entity, enttbl)
+		end
+		
+		if entity.PreEntityCopy then
+			entity:PreEntityCopy()
```

Comment:
Why's this getting called?

Reply:
Wiremod entities add a entry to the table copy which is used for PostEntityPaste. Without this e2 wouldn't execute.